spec/invoice_spec.rb

```diff
@@ -0,0 +1,105 @@
+require 'rspec'
+require './lib/invoice'
+
+describe Invoice do
+  describe '#initialize' do
+    it 'exists' do
+      i = Invoice.new({
+        :id          => 6,
+        :customer_id => 7,
+        :merchant_id => 8,
+        :status      => "pending",
+        :created_at  => Time.now,
+        :updated_at  => Time.now,
+      })
+
+      expect(i).is_a? Invoice
+    end
+
+    it 'has an id' do
+      i = Invoice.new({
+        :id          => 6,
+        :customer_id => 7,
```

Comment:
I think hound will be cranky about this spacing now. I remember we talked about doing it this way but then we jumped over to whatever rubocop does with it

Reply:
@NoahZinter Thanks for the catch!!